src/Cli/dotnet/CommandFactory/CommandResolution/AbstractPathBasedCommandResolver.cs

```diff
@@ -4,51 +4,50 @@
 using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.Cli.Utils.Extensions;
 
-namespace Microsoft.DotNet.CommandFactory
+namespace Microsoft.DotNet.Cli.CommandFactory.CommandResolution;
```

Comment:
Roslyn for example uses Microsoft.DotNetCommandFactory namespace in benchmark test.

https://github.com/dotnet/roslyn/blob/1a2fa4290c83375622f3b3ecf1facedf9d6aff06/src/Tools/IdeBenchmarks/StackTraceExplorer/StackTraceExplorerBenchmarks.cs#L29

Reply:
... Wait wait. They don't take a dependency on this. Instead, they depend on the output of a stack trace? Okay, that's their problem to fix. That's like a really specific loose-coupling to the namespaces/classes of our DLLs.

Reply:
Pinging @dotnet/roslyn-ide just as an FYI that we're changing classes and namespaces and other things for the CLI.